src/persistence/sql/cms_storage_sql_builder.e

```diff
@@ -36,6 +36,7 @@ feature -- Initialization
 			create u.make ("admin")
 			u.set_password ("istrator#")
 			u.set_email (a_setup.site_email)
+			u.set_status ({CMS_USER}.active)
```

Comment:
I guess we should have 
u.activate
or 
u.set_active

but we should not request the dev to use the constants.